src/clr-angular/layout/_grid.clarity.scss

```diff
@@ -7,4 +7,10 @@
         margin-left: 0;
         margin-right: 0;
     }
+
+    // Override grid, helps with issues for popovers and such
+    // See https://github.com/twbs/bootstrap/pull/25255
+    .col-lg, .col-lg-1, .col-lg-2, .col-lg-3, .col-lg-4, .col-lg-5, .col-lg-6, .col-lg-7, .col-lg-8, .col-lg-9, .col-lg-10, .col-lg-11, .col-lg-12, .col-md, .col-md-1, .col-md-2, .col-md-3, .col-md-4, .col-md-5, .col-md-6, .col-md-7, .col-md-8, .col-md-9, .col-md-10, .col-md-11, .col-md-12, .col-sm, .col-sm-1, .col-sm-2, .col-sm-3, .col-sm-4, .col-sm-5, .col-sm-6, .col-sm-7, .col-sm-8, .col-sm-9, .col-sm-10, .col-sm-11, .col-sm-12, .col-xl, .col-xl-1, .col-xl-2, .col-xl-3, .col-xl-4, .col-xl-5, .col-xl-6, .col-xl-7, .col-xl-8, .col-xl-9, .col-xl-10, .col-xl-11, .col-xl-12, .col-xs, .col-xs-1, .col-xs-2, .col-xs-3, .col-xs-4, .col-xs-5, .col-xs-6, .col-xs-7, .col-xs-8, .col-xs-9, .col-xs-10, .col-xs-11, .col-xs-12 {
```

Comment:
I know a few things break with `position: static` but could we just use something like:
https://stackoverflow.com/a/22509502/3538394

instead of writing all the classes down in case we decide to keep this style?

Reply:
Yeah, didn't get fancy because I just copied this from Bootstrap. We need to decide on our solution with grids first.